scipy/stats/tests/test_stats.py

```diff
@@ -1915,12 +1915,22 @@ def test_sem(self):
         assert_raises(ValueError, stats.sem, x, nan_policy='raise')
         assert_raises(ValueError, stats.sem, x, nan_policy='foobar')
 
-    def test_zmap(self):
-        # not in R, so tested by using:
-        #     (testcase[i] - mean(testcase, axis=0)) / sqrt(var(testcase) * 3/4)
-        y = stats.zmap(self.testcase,self.testcase)
-        desired = ([-1.3416407864999, -0.44721359549996, 0.44721359549996, 1.3416407864999])
-        assert_array_almost_equal(desired,y,decimal=12)
+
+class TestZmapZscore:
```

Comment:
I would add some tests that include a `ddof` parameter when there are `nans` and nan_policy is `omit`.

Reply:
There is a basic test of `zscore` with `nan_policy='propagate'` that verifies an input with a `nan` returns an array that is all `nan`.  Do you think it needs an additional test?

Reply:
No, now I see it below.